docs/configuration.md

```diff
@@ -4,7 +4,7 @@
 
 ## ⚙ Plugin Configurations
 
-* `Time Zone`: The time zone your team is working in. This is to make sure all datetimes you enter are interpreted in your timezone and not in server's. This can be overridden in channel standup configuration.
+* `Time Zone`: The time zone your team is working in. This is to make sure all datetimes you enter are interpreted in your timezone and not the server's. This can be overridden in channel standup configuration.
```

Comment:
```suggestion
* `Time Zone`: The time zone your team is working in. This is to make sure all the date times you enter are interpreted in your timezone and not the servers. This can be overridden in channel standup configuration.
```